### DISCUSSION TITLE
how to make a tag same as link in md

### DISCUSSION THREAD

**Discussion by zenkiet** [vuepress/core#412] [Q&A] [answered]
I realized when I used `[name](link)`, good redirect
but when i use a tag html, it's not like that

I need to use the a tag but it has the same function as markdown, is it possible

**Comment by Mister-Hope** [comment, marked as answer]
We already pointed out clearly that link in markdown is renderd as `<RouteLink>`, so if you do insist using tags, try importing `<RouteLink` component and use it.

See https://v2.vuepress.vuejs.org/guide/markdown.html#links
